sdk/appcenter-distribute/src/test/java/com/microsoft/appcenter/distribute/download/http/HttpConnectionCheckTaskTest.java

```diff
@@ -0,0 +1,128 @@
+package com.microsoft.appcenter.distribute.download.http;
```

Comment:
Copyright is missing, you can configure it in Android Studio (check our internal wiki).